src/contracts/addreses.ts

```diff
@@ -113,36 +113,36 @@ const addresses: Record<GebDeployment, ContractList> = {
     'optimism-sepolia': {
         MULTICALL: '0xcA11bde05977b3631167028862bE2a173976CA11',
         ETH: '0x4200000000000000000000000000000000000006',
-        GEB_SYSTEM_COIN: '0xd87Dd8e541BB8027f5d7292b2096a59DCa056C76',
-        GEB_PROTOCOL_TOKEN: '0xbEa0B991DfA52b6935F968fEf4279ba8472326E2',
-        GEB_SAFE_ENGINE: '0x2e7DdAddFa10E0b88fE084e2Cc2Cd8BD6c5d6a98',
-        GEB_ORACLE_RELAYER: '0x1c76E151DB66cD6940934E04e6a497764d335600',
-        GEB_SURPLUS_AUCTION_HOUSE: '0x0df99181289acB1Fc3659B008651eC510c67d342',
```

Comment:
😭

Reply:
solidity `dev` branch should be updated after this PR:
https://github.com/hai-on-op/core/pull/40/files

sorry for this 🙏